src/app/views/sidebar/history/History.tsx

```diff
@@ -406,6 +418,18 @@ export class History extends Component<IHistoryProps, any> {
     }
   }
 
+  private trackHistoryItemEvent = (eventName: string, componentName: string, 
+      query: IHistoryItem) => {
+    telemetry.trackEvent(
+      eventName,
+      {
+        ComponentName: componentName,
+        ItemIndex: query.index,
+        QueryUrl: query.url,
```

Comment:
i think we should remove the url because it can contain user id information in some urls.  we'd need to strip it out but i don't think we necessarily need it so we can just pull it for now.